### ISSUE TITLE
Mining speed slightly slower than tools?

### ISSUE THREAD

**Issue by darthvader45** [rwtema/modjam2018#18]
Just wondering, but with the new version, when I get the mining buffs from certain levels, my hands mine slightly slower than the equivalent tools.  Is this intentional, or a slight error on speed?

**Comment by ghost** [comment]
I'm getting the same thing, but I just watched DW20's FC episode and saw the blocks breaking at the correct speed, so I expect it's a mod interaction. The questions are which mod and what's getting interfered with?

**Comment by darthvader45** [comment]
Here's my mod list:
![image](https://user-images.githubusercontent.com/2816420/44033009-013f0b40-9ed7-11e8-8ba4-42d04ce2b9ed.png)

It seems to be my unarmed mining speed that's getting interfered with, specifically the hand buffs (wooden hands, iron hands, etc.)

**Comment by ghost** [comment]
Same thing here. That's a pretty small modlist. I don't see anything that looks like it would interfere. I've also got Malisis and Rustic, and they're not in the DW20 pack, so they could be causes, but I don't see why.

~~If I could suggest, duplicate your game directory and then, using the copy, look for the conflict.~~

~~On a list this small, I'd deactivate 2 or 3 at a time until you can break stone bare at a faster speed. When you think you've found the mod interacting, work on removing the rest top make sure it's not an interaction.~~

~~If you can get your list down to 1 or 2 mods and you still have the problem...~~

**Comment by ghost** [comment]
So, I built your pack with the list and versions you provided, except for Optifine and while it wasn't instant, I could break blocks quickly.

You might want to try resetting your level with the /monk setlevel command. If you don't have cheats on, you'll need to open your game to LAN to run them. I'm not @rwtema, I don't know him, and I've never seen him post here. I'm just trying to help out.

A couple of things to remember:
1. Optifine messes with a lot of code; many modders won't take bug reports if Optifine is loaded.
2. Most modders won't take reports if you haven't tried updating to the most recent version of the mod. If you need to keep versions for a server, you can still duplicate the instance and test with that.

If you can give detailed instructions to replicate the issue (the version specific mod list was a good start) it's more likely to get fixed because others can see what happened.

**Comment by darthvader45** [comment]
Yeah, I at first thought it was something in the config, as the speeds
there were set noticeably lower than what I was expecting.  I'll make sure
to stay updated, and maybe this issue will resolve itself.
